lib/cdo/lesson_import_helper.rb

```diff
@@ -327,14 +310,14 @@ def self.find_tips(markdown)
     markdown.to_enum(:scan, regex).map {Regexp.last_match}
   end
 
-  def self.find_tip_links(markdown)
+  def self.find_first_tip_link(markdown)
     # See https://github.com/code-dot-org/remark-plugins/blob/master/src/tiplink.js
     # Looks for the location where tip icons should appear
     # Example: tip!!!tip-0<!-- place where you'd like the icon --> some text
     # <!-- place where you'd like the icon --> is optional but is written out
     # in this regex in order to be able to correctly the text that should be displayed
-    regex = /^([\w-]+)!!! ?([\w-]+)(?:<!-- place where you'd like the icon -->)?(.*\n?.*)$/
-    markdown.to_enum(:scan, regex).map {Regexp.last_match}
+    regex = /^?(\S+ +)?([\w-]+)!!! ?([\w-]+)(?:<!-- place where you'd like the icon -->)?(.*\n?.*)$/
```

Comment:
This change allows (optionally) for some non-whitespace characters followed by at least one space to come before tip icons.